apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/ExperimentsResourceTest.java

```diff
@@ -1626,6 +1627,87 @@ void getById() {
         }
     }
 
+    @Nested
+    @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+    class DeleteExperimentItems {
+
+        @Test
+        void deleteExperimentsById__whenExperimentExists__thenReturnNoContent() {
+            var experiment = podamFactory.manufacturePojo(Experiment.class);
+
+            createAndAssert(experiment, API_KEY, TEST_WORKSPACE);
+
+            deleteExperimentAndAssert(Set.of(experiment.id()), API_KEY, TEST_WORKSPACE);
+        }
+
+        private void deleteExperimentAndAssert(Set<UUID> ids, String apiKey, String workspaceName) {
+            try (var actualResponse = client.target(getExperimentsPath())
+                    .path("delete")
+                    .request()
+                    .header(HttpHeaders.AUTHORIZATION, apiKey)
+                    .header(WORKSPACE_HEADER, workspaceName)
+                    .post(Entity.json(new ExperimentsDelete(ids)))) {
+
+                assertThat(actualResponse.getStatusInfo().getStatusCode()).isEqualTo(204);
+            }
+
+            ids.parallelStream().forEach(id -> getExperimentAndAssertNotFound(id, apiKey, workspaceName));
+        }
+
+        private void getExperimentAndAssertNotFound(UUID id, String apiKey, String workspaceName) {
+            try (var actualResponse = client.target(getExperimentsPath())
+                    .path(id.toString())
+                    .request()
+                    .header(HttpHeaders.AUTHORIZATION, apiKey)
+                    .header(WORKSPACE_HEADER, workspaceName)
+                    .get()) {
+
+                assertThat(actualResponse.getStatusInfo().getStatusCode()).isEqualTo(404);
+            }
+        }
+
+        @Test
+        void deleteExperimentsById__whenExperimentDoesNotExist__thenReturnNoContent() {
+            var experiment = podamFactory.manufacturePojo(Experiment.class);
+
+            getExperimentAndAssertNotFound(experiment.id(), API_KEY, TEST_WORKSPACE);
+
+            deleteExperimentAndAssert(Set.of(experiment.id()), API_KEY, TEST_WORKSPACE);
+        }
+
+        @Test
+        void deleteExperimentsById__whenExperimentHasItems__thenReturnNoContent() {
+            var experiment = podamFactory.manufacturePojo(Experiment.class);
+
+            createAndAssert(experiment, API_KEY, TEST_WORKSPACE);
+
+            var experimentItems = PodamFactoryUtils.manufacturePojoList(podamFactory, ExperimentItem.class).stream()
+                    .map(experimentItem -> experimentItem.toBuilder().experimentId(experiment.id()).build())
+                    .collect(Collectors.toUnmodifiableSet());
+
+            var createRequest = ExperimentItemsBatch.builder().experimentItems(experimentItems).build();
+
+            createAndAssert(createRequest, API_KEY, TEST_WORKSPACE);
+
+            deleteExperimentAndAssert(Set.of(experiment.id()), API_KEY, TEST_WORKSPACE);
```

Comment:
Minor: you should probably test this case for multiple experiments with multiple experiment ids.